relay-server/src/actors/processor.rs

```diff
@@ -2294,8 +2296,21 @@ impl EnvelopeProcessorService {
                     let Some(span_description) = inner_span.description.value() else {
                         continue;
                     };
+                    let span_system: &str = inner_span
+                        .data
+                        .value()
+                        .and_then(|v| v.get("span.system"))
+                        .and_then(|system| system.as_str())
+                        .unwrap_or_default();
                     if all_modules_enabled
-                        || span_op.starts_with("db") && !span_description.contains(r#""$"#)
+                        || (span_op.starts_with("db")
+                            && op_denylist
+                                .clone()
+                                .into_iter()
+                                .any(|op| span_op.contains(op))
+                            && !(span_op == "db.sql.query"
+                                && !(span_description.contains(r#""$"#)
+                                    || span_system == "mongodb")))
```

Comment:
we should probably string enum all of these sometime soon given there will be more coming.

Reply:
I'll refactor when we start having more. Maybe we'll fix them fast and we won't need that.